src/components/common/view/TopBar.js

```diff
@@ -34,4 +36,12 @@ const TopBar = ({ hideBalance, push, children }) => {
   )
 }
 
+const styles = StyleSheet.create({
+  topBar: {
+    paddingTop: normalize(8),
+    paddingBottom: normalize(8),
+    marginBottom: normalize(8)
```

Comment:
I think 8 is the default value for margins and paddings, so in the same way than other comments, we should use theme value here